src/object/map-values/map-values.ts

```diff
@@ -1,7 +1,26 @@
 import { entriesOf } from '../entries-of'
 import { fromEntries } from '../from-entries'
 
-export function mapValues<T extends ArrayLike<unknown> | {}, Mapper extends (v: T[keyof T], k: keyof T) => unknown>(
+/**
+ * Returns the names of the enumerable string properties and methods of an object.
+ *
+ * ### Example
+ * ```ts
+ * keysOf([["foo", "bar"]])
+ * // => ["foo"]
+ *
+ * keysOf({})
+ * // => []
+ * ```
+ *
+ * ### Alternatives
+ * - [Lodash - keys](https://lodash.com/docs/4.17.15#keys)
+ *
+ * @param obj - The object to get the keys from.
+ * @returns The keys as an array.
+ *
+ * @group Object
+ */ export function mapValues<T extends ArrayLike<unknown> | {}, Mapper extends (v: T[keyof T], k: keyof T) => unknown>(
```

Comment:
```suggestion
/**
 * Returns an object with same keys, where the original values are converted using a mapper.
 *
 * ### Example
 * ```ts
 * mapValues({ foo: 0, bar: 2 }, (x) => x + 1)
 * // => { foo: 1, bar: 3 }
 * ```
 *
 * ### Alternatives
 * - [Lodash - mapValues](https://lodash.com/docs/4.17.15#mapValues)
 *
 * @param obj - The object to map the values from.
 * @returns The keys as an array.
 *
 * @group Object
 */
 export function mapValues<T extends ArrayLike<unknown> | {}, Mapper extends (v: T[keyof T], k: keyof T) => unknown>(
```